File: test/Enum.test.ts
import {expect} from 'chai';
import {DayOfWeek} from "../src/enum/DayOfWeek";

describe('Enum test', () => {
  it('It should return true', () => {
    expect(DayOfWeek.isWeekendDay(DayOfWeek.SUNDAY)).to.be.true;
  });
  
  it('It should return false', () => {
    expect(DayOfWeek.isWeekendDay(DayOfWeek.FRIDAY)).to.be.false;
  });
});

File: test/src/EnumTest.kt
import enum.DayOfWeek
import org.testng.AssertJUnit.assertEquals
import org.testng.annotations.Test

class EnumTest{

  @Test
  fun `It should return true`(){
    assertEquals(DayOfWeek.isWeekendDay(DayOfWeek.SUNDAY), true)
  }

  @Test
  fun `It should return false`(){
    assertEquals(DayOfWeek.isWeekendDay(DayOfWeek.FRIDAY), false)
  }
}